subiquity/common/filesystem/tests/test_manipulator.py

```diff
@@ -91,6 +92,25 @@ def test_delete_encrypted_vg(self):
             a for a in manipulator.model._actions if a.type == 'dm_crypt']
         self.assertEqual(dm_crypts, [])
 
+    def test_wipe_existing_fs(self):
+        # LP: #1983036 - edit a partition to wipe it and mount it, but not
+        # actually change the fs type already there
+        # example: it's ext4, wipe and make it ext4 again
+        manipulator, d = make_manipulator_and_disk()
+        p = make_partition(manipulator.model, d, preserve=True)
+        fs = make_filesystem(manipulator.model, partition=p, fstype='ext2',
+                             preserve=True)
+        manipulator.model._actions.append(fs)
+        manipulator.delete_filesystem(fs)
+        spec = {'wipe': 'random'}
```

Comment:
I think to match our discussion, this spec should have 'fstype': 'ext2'?

Reply:
Comment matches the code now.  I did mean for it to say 'ext2', as the default is 'ext4' I wanted to use a different value in the test.